### PR TITLE
PMM-3581 Query examples.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-3581

### CODE REVIEW

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-326/

File: api/qanpb/object_details.proto [hunk 1]
```diff
@@ -65,5 +74,35 @@ message MetricValues {
...
+message Labels {
+  repeated string value = 1;
+}
+
+// QueryExampleRequest defines filtering of query examples for specific value of
+// dimention (ex.: host=hostname1 or queryid=1D410B4BE5060972.
+message QueryExampleRequest {
+  google.protobuf.Timestamp period_start_from = 1;
+  google.protobuf.Timestamp period_start_to = 2;
+  string filter_by = 3; // dimention: queryid | host ...
```
Comment (AlekSi) [inline api/qanpb/object_details.proto]: Please write comments above fields. That way they show up in generated code and Swagger docs.

Comment (askomorokhov) [reply api/qanpb/object_details.proto]: Fixed.

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-326/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-326/

**Pull request merged**